ci/svd2rust-regress/README.md

```diff
@@ -40,36 +40,11 @@ If all test cases passed, the return code will be `0`. If any test cases failed,
 
 ### Options
 
-Here are the options for running `svd2rust-regress`:
-
+You can display options for `svd2rust-regress` by running:
 
 ```text
-svd2rust-regress 0.1.0
-James Munns <james.munns@gmail.com>:The svd2rust developers
-
-USAGE:
-    svd2rust-regress [FLAGS] [OPTIONS]
-
-FLAGS:
-    -b, --bad-tests    Include tests expected to fail (will cause a non-zero return code)
-    -f, --format       Enable formatting with `rustfmt`
-    -h, --help         Prints help information
-    -l, --long-test    Run a long test (it's very long)
-    -V, --version      Prints version information
-    -v, --verbose      Use verbose output
-
-OPTIONS:
-    -a, --architecture <arch>
-            Filter by architecture, case sensitive, may be combined with other filters Options are: "CortexM", "RiscV", "Msp430", "Mips" and "XtensaLX"
-    -p, --svd2rust-path <bin_path>
-            Path to an `svd2rust` binary, relative or absolute. Defaults to `target/release/svd2rust[.exe]` of this
-            repository (which must be already built)
-    -c, --chip <chip>                            Filter by chip name, case sensitive, may be combined with other filters
-    -m, --manufacturer <mfgr>
-            Filter by manufacturer, case sensitive, may be combined with other filters
-
-        --rustfmt_bin_path <rustfmt_bin_path>
-            Path to an `rustfmt` binary, relative or absolute. Defaults to `$(rustup which rustfmt)`
+# in the ci/svd2rust-regress folder
+cargo run --release -- help
```

Comment:
this is unnecessary, you can just do `cargo run -p svd2rust-regress` or, even simpler, use [our alias](https://github.com/rust-embedded/svd2rust/blob/0e2aa424e0033c5b21b8ddec780db4af81959d1e/.cargo/config.toml#L5) `cargo regress`

specifying `--release` is not necessary

```suggestion
cargo regress --help
```

Reply:
done